lang/nl.js

```diff
@@ -48,10 +49,11 @@ export default {
         vereniging kan vertellen en je kan voorstellen aan wat mensen.`
     },
     bar_buddy: {
-      title: 'Barbuddy Systeem',
-      description: `Wil jij een keertje langskomen op een baravond maar ben je bang dat je je niet meteen op je gemak
-        voelt? Meld je dan aan voor een barbuddy! Een barbuddy is iemand die jou de ins en outs kan vertellen over de 
-        vereniging, maar ook iemand die jou voor kan stellen aan nieuwe mensen.`,
+      title: `Barbuddy's`,
+      description: `Wil jij een keertje langskomen op een baravond maar ben je nog niet bekend met de vereniging? Meld
+      je dan aan voor een barbuddy! Een barbuddy is iemand die van tevoren met je afspreekt zodat je samen naar de
```

Comment:
Can we keep the indentation here? :)

Reply:
Sure, but there are many other places where this indentation is inconsistent. Fixing all of them I consider out of scope.